If the repository is not git repository, it throws exception at start up

```
[  89429]  ERROR - tartup.impl.StartupManagerImpl - fatal: not a git repository (or any of the parent directories): .git
 during executing git log --merges --pretty=format:%x01%H%x02%ct%x02%an%x02%at%x02%ae%x02%cn%x02%ce%x02%P%x02%s%x02%b%x02%B%x03 --encoding=UTF-8 -M --name-status -c -- 
com.intellij.openapi.vcs.VcsException: fatal: not a git repository (or any of the parent directories): .git
 during executing git log --merges --pretty=format:%x01%H%x02%ct%x02%an%x02%at%x02%ae%x02%cn%x02%ce%x02%P%x02%s%x02%b%x02%B%x03 --encoding=UTF-8 -M --name-status -c --
	at git4idea.commands.GitSimpleHandler.run(GitSimpleHandler.java:241)
	at git4idea.history.GitHistoryUtils.loadDetails(GitHistoryUtils.java:843)
	at git4idea.history.GitHistoryUtils.history(GitHistoryUtils.java:806)
	at com.github.shiraji.findpullrequest.model.GitHistoryService.findLatestMergeCommit(GitHistoryService.kt:13)
	at com.github.shiraji.findpullrequest.activity.DetectHostingServiceStartupActivity.runActivity(DetectHostingServiceStartupActivity.kt:13)
	at com.intellij.ide.startup.impl.StartupManagerImpl$2.run(StartupManagerImpl.java:147)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runActivity(StartupManagerImpl.java:342)
	at com.intellij.ide.startup.impl.StartupManagerImpl.access$700(StartupManagerImpl.java:58)
	at com.intellij.ide.startup.impl.StartupManagerImpl$3.run(StartupManagerImpl.java:165)
	at com.intellij.openapi.project.DumbServiceImpl.updateFinished(DumbServiceImpl.java:318)
	at com.intellij.openapi.project.DumbServiceImpl.access$1100(DumbServiceImpl.java:52)
	at com.intellij.openapi.project.DumbServiceImpl$10.run(DumbServiceImpl.java:503)
	at com.intellij.openapi.application.impl.LaterInvocator$1.run(LaterInvocator.java:158)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:337)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:321)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:866)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:654)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:381)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: com.intellij.openapi.vcs.VcsException: fatal: not a git repository (or any of the parent directories): .git

	at git4idea.commands.GitSimpleHandler$1.processTerminated(GitSimpleHandler.java:227)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.util.EventDispatcher.dispatch(EventDispatcher.java:88)
	at com.intellij.util.EventDispatcher.access$100(EventDispatcher.java:34)
	at com.intellij.util.EventDispatcher$1.invoke(EventDispatcher.java:68)
	at com.sun.proxy.$Proxy82.processTerminated(Unknown Source)
	at git4idea.commands.GitTextHandler$1.processTerminated(GitTextHandler.java:83)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.execution.process.ProcessHandler$4.invoke(ProcessHandler.java:214)
	at com.sun.proxy.$Proxy14.processTerminated(Unknown Source)
	at com.intellij.execution.process.ProcessHandler$3.run(ProcessHandler.java:171)
	at com.intellij.execution.process.ProcessHandler$TasksRunner.execute(ProcessHandler.java:247)
	at com.intellij.execution.process.ProcessHandler.notifyTerminated(ProcessHandler.java:153)
	at com.intellij.execution.process.ProcessHandler.notifyProcessTerminated(ProcessHandler.java:149)
	at com.intellij.execution.process.BaseOSProcessHandler.onOSProcessTerminated(BaseOSProcessHandler.java:150)
	at com.intellij.execution.process.BaseOSProcessHandler$1$1.consume(BaseOSProcessHandler.java:115)
	at com.intellij.execution.process.BaseOSProcessHandler$1$1.consume(BaseOSProcessHandler.java:100)
	at com.intellij.execution.process.ProcessWaitFor$1.run(ProcessWaitFor.java:55)
	at com.intellij.openapi.application.impl.ApplicationImpl$8.run(ApplicationImpl.java:366)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

--- a/src/main/kotlin/com/github/shiraji/findpullrequest/activity/DetectHostingServiceStartupActivity.kt
+++ b/src/main/kotlin/com/github/shiraji/findpullrequest/activity/DetectHostingServiceStartupActivity.kt
@@ -4,14 +4,16 @@ import com.github.shiraji.findpullrequest.model.*
 import com.intellij.ide.util.PropertiesComponent
 import com.intellij.openapi.project.Project
 import com.intellij.openapi.startup.StartupActivity
+import git4idea.GitUtil
 
 class DetectHostingServiceStartupActivity : StartupActivity {
     override fun runActivity(project: Project) {
+        val config = PropertiesComponent.getInstance(project)
+        if (config.hasHosting()) return
+        if (GitUtil.getRepositoryManager(project).repositories.isEmpty()) return
         val gitHistoryService = GitHistoryService()
         val mergeCommit = gitHistoryService.findLatestMergeCommit(project)
-        val pair = gitHistoryService.findPrNumberAndHostingService(mergeCommit)
-        val hostingService = pair.second ?: return
-        val config = PropertiesComponent.getInstance(project)
+        val hostingService = gitHistoryService.findPrNumberAndHostingService(mergeCommit).second ?: return
         config.setHosting(hostingService)
     }
 }

